ui/src/RecentFilesDrawer/ItemsList.tsx

```diff
@@ -0,0 +1,99 @@
+import { DragEvent, useContext, useEffect, useState } from "react";
+import { RecentFilesContext } from "../WorkspaceContext";
+import {
+  foldersTable,
+  isFolder,
+  workflowsTable,
+} from "../db-tables/WorkspaceDB";
+import { Folder, Workflow } from "../types/dbTypes";
+import FilesListFolderItem from "./FilesListFolderItem";
+import WorkflowListItem from "./WorkflowListItem";
+import { Box } from "@chakra-ui/react";
+import { folderOnTopLocalStorageKey } from "./types";
+
+export default function ItemsList({
+  items,
+}: {
+  items: Array<Folder | Workflow>;
+}) {
+  const [folderOnTop, setFolderOnTop] = useState(
+    window.localStorage.getItem(folderOnTopLocalStorageKey) === "true",
+  );

```

Comment:
you shouldn't store/get settings from localstorage, put it into a field in userSettings folder